Assets/_PlatformerDevelopment/Scripts/Player/PlayerJumpBehaviour.cs

```diff
@@ -0,0 +1,63 @@
+﻿using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace PersonalDevelopment
+{
+    [RequireComponent(typeof(PlayerInput))]
+    [RequireComponent(typeof(Rigidbody))]
+    public class PlayerJumpBehaviour : MonoBehaviour
+    {
+        private const string FloorTag = "Floor";
+        
+        // Components
+        private PlayerCharacterBindings _bindings = null;
+        private Rigidbody _rigidbody = null;
+        
+        // Fields
+        [SerializeField] private float _jumpForce = 10f;
+        private bool _canJump = true;
+
+        #region Movement
+
+        private void OnJumpPressed(InputAction.CallbackContext context)
+        {
+            var pressedUp = context.ReadValue<Vector2>().y > 0;
```

Comment:
After thinking about it, I think it would be more "correct" if we make the movement a 1-d axis composite (instead of a 2d vector) and create another binding button for jump. This would make it so that if/when we want to use a controller instead, the bindings can easily be made

Reply:
Also this would eliminate the unnecessary check to see if the y value is above 0 or not

Reply:
Yes you're right. I just had a fuck it mentality when writing this. I would make jump
W, up arrow and space bar?

Reply:
Yes you're right. I just had a fuck it mentality when writing this. I would make jump
W, up arrow and space bar?

Reply:
hmmm W seems good. Actually I think you can bind both W and spacebar at the same time, if you want!

Reply:
Yes, I was thinking of binding all 3 at the same time

Reply:
Actually removing spacebar, because won't make sense with 2 players